src/views/index.njk

```diff
@@ -1,21 +1,153 @@
+            <h2>Welcome to the new social web</h2>
+        </div>
+        <div class="flex">
+            <div class="col4">
+                <div class="tile valign-wrapper">
+                4 <strong>Projects</strong>
+                </div>
+            </div>
+            <div class="col4">
+                <div class="tile valign-wrapper">
+                {{ data.totals.pod_count }} <strong>Nodes</strong>
+                </div>
+            </div>
+            <div class="col4">
+                <div class="tile valign-wrapper">
+                {{ data.totals.total_users }} <strong>Users</strong>
+                </div>
+            </div>
+            <div class="col4">
+                <div class="tile valign-wrapper">
+                ONE <strong>Network</strong>
+                </div>
+            </div>
+        </div>
+    </header>
 
-    {% include "_network_stats.njk" %}
-    {% include "_node_list.njk" %}
-    {% include "_info_the_federation.njk" %}
-    {% include "_info_site.njk" %}
-
-    <p class="text-center">
-        built with &lt;3 by <a href="https://jasonrobinson.me">jason robinson</a> and <a href="https://github.com/jaywink/the-federation.info/graphs/contributors">contributors</a>
-    </p>
+    <section class="tile">
+        <header>
+            <h2>What is The Federation?</h2>
+        </header>
+        <div>
+            <div class="flex">
+                <div class="col2">
+                    <p>
+                        The Federation refers to a global social network composed by nodes talking together.
+                        Each of them is an installation of a software which supports the <a href="https://github.com/SuperTux88/diaspora_federation">diaspora* protocol</a>.
+                        This website lists all the registered servers among the world to draw statistics. You know, we love numbers and numbers love us.
+                    </p>
+                    <div class="center">
+                        <a href="/global-stats.html" class="btn btn-primary">Visit network evolution</a>
+                    </div>
+                </div>
+                <div class="col2">
+                    <ul>
+                        <li>Nodes: <strong>{{ data.totals.pod_count }}</strong></li>
+                        <li>Users: <strong>{{ data.totals.total_users }}</strong></li>
+                        <li>Last 6 months active users: <strong>{{ data.totals.active_users_halfyear }}</strong></li>
+                        <li>Last month active users: <strong>{{ data.totals.active_users_monthly }}</strong></li>
+                        <li>Posts: <strong>{{ data.totals.local_posts }}</strong></li>
+                        <li>Comments: <strong>{{ data.totals.local_comments }}</strong></li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="tile">
+        <header>
+            <h2>diaspora* - Friendica - Redmatrix - Hubzilla</h2>
```

Comment:
Redmatrix didn't "die", it just evolved into Hubzilla. Redmatrix AFAIK only exists because not all redmatrix nodes have migrated to hubzilla. Maybe they never will, but afaik the software is not developed.

The repo even says

> This project has been renamed and the repository remains only for historical context. Please see https://github.com/redmatrix/hubzilla . 

We do have a problem though with tying The Federation down to certain server softwares. Forks are bound to happen (for example BlaBlaNet, see [this issue](https://github.com/jaywink/the-federation.info/issues/58), also new software comes up, for example my [Socialhome](https://github.com/jaywink/socialhome). We should allow more variants. Maybe if we showcase the top three (diaspora, friendica, hubzilla) - and then ignore the rest for now and later add so that they are dynamically added from the `pods` table? If the server software becomes bigger, we add a separate page for it.